scripts/updateUID.Dockerfile

```diff
@@ -26,7 +27,7 @@ RUN eval $(sed -n "s/${REMOTE_USER}:[^:]*:\([^:]*\):\([^:]*\):[^:]*:\([^:]*\).*/
 		if [ "$OLD_GID" != "$NEW_GID" ]; then \
 			sed -i -e "s/\([^:]*:[^:]*:\)${OLD_GID}:/\1${NEW_GID}:/" /etc/group; \
 		fi; \
-		chown -R $NEW_UID:$NEW_GID $HOME_FOLDER; \
+	find $HOME_FOLDER -path IGNORE_PATTERN -prune -o -print0 | xargs -0 chown $NEW_UID:$NEW_GID; \
```

Comment:
Missing `$` in front of `IGNORE_PATTERN`.

Reply:
Added! Thanks!